src/components/AllJS/books.js

```diff
@@ -22,8 +23,8 @@ const Books = () => {
     if (titlecheck.value === '' || authorcheck.value === '') {
       alert('Please fill the field');
     } else {
+      api.addNewBook(book);
```

Comment:
- [ ] @RashidMenhas great work on your component ,however this method is suppose to be in the reducer similar to the delete and getbooks methods.